.github/workflows/test-node.js.yml

```diff
@@ -16,7 +16,11 @@ jobs:
 
     strategy:
       matrix:
-        node-version: [10.x, 12.x, 14.x]
+        node-version:
+        - 10.x
+        - 12.x
+        - 14.x
+        - 15.x
```

Comment:
I am now thinking we can leave Node.js 15 out since initial release of Node.js 16 should be available in the next few days: https://nodejs.org/en/about/releases/

```suggestion
```

Reply:
So let's add 16 when it's ready?

Reply:
Yes. Should I wait for 16 before merging this one?